src/debug/recorder_debug.cpp

```diff
@@ -44,7 +46,7 @@ const Menu_t Recorder_debug_menu[] =
     {9, "Pop Last Packet", &REC_testPopLastPacket, MENU_CMD},
     {0, nullptr, nullptr, MENU_NULL}
 };
-
+/// @cond HELPER FUNCTIONS
```

Comment:
This is not an acceptable solution.  All of these should be documented.

Reply:
Can I add the functions to the header file and document there, or should I document them directly in the .cpp file?

Reply:
If these functions are used outside of this translation unit, then move the declarations to the header file and document there. Otherwise, apply static linkage to these functions and document in this file.